changes/16840-db-changes-for-script-limit-increase

```diff
@@ -1 +1,2 @@
 * Updated the database schema to support the increase in size of scripts.
+* **NOTE**: database migration might take a few minutes to complete, depending on scripts usage. It is mostly affected by the number of script executions (anonymous scripts via `fleetctl run-script` or saved scripts), the number of hosts targeted to execute the scripts and the number of saved scripts.
```

Comment:
@lukeheath Just want to bring this to your attention for the release notes. I did run a load test locally and for 5000 hosts, 100 (big) saved scripts, one execution of every script on every host, and an additional 100 anonymous script execution (1 host, 1 anonymous script), the migration took ~1m30 on my machine.

I _think_ that's a lot of executions. I don't think there can be too many anonymous scripts execution, as `fleetctl run-script` targets a single host at a time, but the saved scripts can easily target a lot of hosts, and if those saved scripts are edited often (especially easy with the gitops approach to have them re-uploaded as needed) and then run regularly, there could be many more executions of different script contents than that. Let me know if you want me to tweak the wording.

Reply:
Thanks for the heads up! Changefile looks good - I'll call it out prominently in the release notes.